src/satellite/process.ts

```diff
@@ -25,7 +25,8 @@ import {
 } from '../util/types'
 import { Satellite } from './index'
 import { SatelliteConfig, SatelliteOpts } from './config'
-import { mergeChangesLastWriteWins, mergeOpTypesAddWins } from './merge'
+import { mergeChangesLastWriteWins, mergeOpTags } from './merge'
```

Comment:
In this function, couldn't we take advantage of the source to fix the issue with timestamp comparison?
(it seems it still has the issue when timestamps are equal)

Reply:
yes, that was the plan originally, fixed